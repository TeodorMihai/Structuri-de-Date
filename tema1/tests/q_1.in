1 1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::w 0 0 AAAA
1
::w 0 1 AAAA
1
::w 0 2 AAAA
1
::w 0 3 AAAA
1
::w 0 4 AAAA
1
::w 0 5 AAAA
1
::w 0 6 AAAA
1
::w 0 7 AAAA
1
::w 0 8 AAAA
1
::w 0 9 AAAA
1
::w 0 10 AAAA
1
::w 0 11 AAAA
1
::w 0 12 AAAA
1
::w 0 13 AAAA
1
::w 0 14 AAAA
1
::w 0 15 AAAA
1
::r 0 4
1
::r 0 3
1
::r 0 2
1
::r 0 1
1
::w 0 0 BBBB
1
::r 0 1
1
::r 0 0
10
::w 0 10 CCCC
8000
::r 0 2
1 
::w 0 9 DDDD
1
::r 0 9
1
::w 0 9 EEEE
1
::r 0 9
1
::e
