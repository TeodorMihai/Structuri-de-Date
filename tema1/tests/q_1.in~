1 2
::r 0 4
5
::e