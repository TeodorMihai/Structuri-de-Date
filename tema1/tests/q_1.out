AAAA
AAAA
AAAA
AAAA
AAAA
BBBB
AAAA
DDDD
EEEE
375.50 360.50 2320.25 358.00
